### PR TITLE
WEBEU2 - HTTP AJAX - Wasiu Idowu 🇳🇬

### PR DESCRIPTION
**Pull request by Hoxtygen**

### CODE REVIEW

File: makdown.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+function markdownTables(arr) {
```
Comment (maximesalomon) [inline makdown.js:1]: Nice :)

File: friends/src/components/NewFriend.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    const name = event.target.name;
+    const value = event.target.value;
+    this.setState({
+      [name]: value
+    })
+  }
+
+  handleAddFriend = (e) => {
+      //e.preventDefault()
+      axios.post("http://localhost:5000/friends/", {
```
Comment (maximesalomon) [inline friends/src/components/NewFriend.js:67]: Love Axios!

**Review by maximesalomon — COMMENTED** [review]
Hello 👋 

Seems like you have a very good understanding of HTTP Request/Ajax. You used Axios well 👍 

**Feedbacks**
- By putting your styled components above the main logic in your components, it's hard to quickly go through your code and see what's you are trying to do. You could put it below your logic and before the export to help with that.
- Try to use ES6 arrow function whenever you can =>

Overall great job on this project Wasiu 🔥

**Comment by Hoxtygen** [comment]
> Hello 👋
> 
> Seems like you have a very good understanding of HTTP Request/Ajax. You used Axios well 👍
> 
> **Feedbacks**
> 
> * By putting your styled components above the main logic in your components, it's hard to quickly go through your code and see what's you are trying to do. You could put it below your logic and before the export to help with that.
> * Try to use ES6 arrow function whenever you can =>
> 
> Overall great job on this project Wasiu 🔥

Oh, wasn't paying attention to how where the styled components is placed could affect readability. Will refactor and put them down below just before the export.

File: makdown.js (hunk 1 above)
Comment (Hoxtygen) [reply makdown.js:1]: Oh, this wasn't supposed to make it in here. I must have forgotten.

**Comment by maximesalomon** [comment]
Awesome! It's for you as well :)

**Pull request merged**